account_asset_management_xls/report/account_asset_report_xls.py

```diff
@@ -0,0 +1,832 @@
+# Copyright 2009-2018 Noviat
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import models, _
+from odoo.exceptions import UserError
+from odoo.tools.translate import translate
+
+_logger = logging.getLogger(__name__)
+
+
+IR_TRANSLATION_NAME = 'account.asset.report'
+
+
+class AssetReportXlsx(models.AbstractModel):
+    _name = 'report.account_asset_management_xls.asset_report_xls'
+    _inherit = 'report.report_xlsx.abstract'
+
+    def _(self, src):
+        lang = self.env.context.get('lang', 'en_US')
+        val = translate(
```

Comment:
Could it be better if the `self.env['ir.translation']._get_source` was used here as adviced on https://github.com/OCA/OCB/blob/12.0/odoo/tools/translate.py#L336 so that you can avoid passing the cursor that way?